tests/unit/pages/Editor.spec.js

```diff
@@ -1476,6 +1418,9 @@ describe("Editor.vue", () => {
     const itemEditorWrapper = wrapper.findComponent(ItemEditor);
     const inputTextWrapper = itemEditorWrapper.findAllComponents(InputText)[4];
 
+    // clear past values of dialog description
+    await store.dispatch("dialog/unsetDialogDescription");
```

Comment:
no need of await here
- Also, check and change in the codebase wherever non async store action dispatches are written using await